libs/storage/Tsavorite/cs/Tsavorite.sln

```diff
@@ -29,12 +29,12 @@ Global
 		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{33A732D1-2B58-4FEE-9696-B9483496229F}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{33A732D1-2B58-4FEE-9696-B9483496229F}.Debug|Any CPU.Build.0 = Debug|x64
+		{33A732D1-2B58-4FEE-9696-B9483496229F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
is this change to Any CPU intentional?

Reply:
Yes; we previously built x64 before an earlier csproj reorganization PR. That was a good change so we now always build AnyCPU, but it revealed that this was broken (it's why we were seeing that "Configuration incorrect" message when loading Tsavorite.sln)